plugins/balloontoolbar/plugin.js

```diff
@@ -704,7 +715,9 @@
 				this.parts.content.unselectable();
 				CKEDITOR.tools.array.forEach( this.parts.content.find( 'a' ).toArray(), function( element ) {
 					element.setAttribute( 'draggable', 'false' );
-				} );
+					this.registerFocusable( element );
+					this._focusablesItems.push( element );
```

Comment:
Actually we could solve this issue without introducing `this._focusablesItems` - which duplicates [`focusables`](https://docs.ckeditor.com/ckeditor4/docs/#!/api/CKEDITOR.ui.balloonPanel-property-focusables) property. And this is the biggest problem introduced by this approach, we have 2 duplicated places where we keep this information.

Instead we should reuse and rely on `focusables`. The only place where we really need to do something custom is here:

https://github.com/ckeditor/ckeditor-dev/blob/d567e2e45a6ff206c36753a1c3978689ba26cb39/plugins/balloontoolbar/plugin.js#L691-L692

We have to remove focusables that are within `content` part.

To do that we could simply iterate over it and test each focusable if it is a child of content part and remove it if it actually is (do this check for each focusable). And we won't have to worry about logic duplication.